### DISCUSSION TITLE
Is there an official Storage/Local DB strategy?

### DISCUSSION THREAD

**Discussion by kazzkiq** [wailsapp/wails#2598] [Q&A]
Is there any storage or local DB pattern we should follow? As I couldn't find anything related in the docs I decided to ask here.

Also, wouldn't it be useful if Wails exposed a JS API for easy data persistence?

What packages are you guys using for this goal?

**Comment by leaanthony** [comment]
There isn't an official one, but v3 does have plugin support and there is both a sqlite and json db storage plugin, so at least you have something to look forward to in the future :-)
Honestly, just a simple get/set binding that writes to a file is probably all you need.

**Comment by marcio199226** [comment]
personally I'm using https://github.com/nutsdb/nutsdb

**Comment by airtonix** [comment]
I'm shamelessy copying this: 

- https://github.com/Splode/optimus/blob/main/backend/localstore/localstore.go
- https://github.com/Splode/optimus/blob/main/backend/config/config.go

👍🏻
